### PR TITLE
OPSEXP-2149 Refresh docs

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2149

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,62 @@
...
-* [Quay](https://quay.io/repository/alfresco/alfresco-base-java) image name: `quay.io/alfresco/alfresco-base-java`
+The images are available on:
 
-Example final image: `alfresco/alfresco-base-java:jre11-centos7`
+* [Docker Hub](https://hub.docker.com/r/alfresco/alfresco-base-java), image name: `alfresco/alfresco-base-java`
+* [Quay](https://quay.io/repository/alfresco/alfresco-base-java) (enterprise credentials required), image name: `quay.io/alfresco/alfresco-base-java`
 
-> If you are using this base image in a public repository, please stick to the DockerHub published image.
+> If you are using this base image in a public repository, please use the Docker
+> Hub hosted one
```
Comment (alxgomz) [inline README.md:29]: Not sure what we meant here...

File: README.md [hunk 2]
```diff
@@ -1,64 +1,62 @@
...
 
 ### Image pinning
 
-These tags get overwritten to always have an up-to-date image and hopefully
-without security issues.
+All the supported tags are mutable because they are periodically rebuilt, to
+always have an up-to-date image without security issues.
```
Comment (alxgomz) [inline README.md:34]: Sounds like a commitment, please rephrase

File: README.md [hunk 3]
```diff
@@ -1,64 +1,62 @@
...
 
-For being in control of the image update process, you can use the digest in
-addition to the tag in your `Dockerfile`, for example:
+The suggested approach is to pin the sha256 digest for best reproducibility in
+your `Dockerfile`, for example:
 
 ```dockerfile
-FROM alfresco/alfresco-base-java:jre11-centos7@sha256:59a453e01fd958a3748a2e9b0ca99cdf3410f98eeb245499c7bb31696e35bdf4
+FROM alfresco/alfresco-base-java:re17-rockylinux9@sha256:b749868ceb42bd6f58ae2f143e8c16af4752fad7b40eb1085c014cbfcecb1ffc
```
Comment (alxgomz) [inline README.md:40]:
```suggestion
FROM alfresco/alfresco-base-java:jre17-rockylinux9@sha256:b749868ceb42bd6f58ae2f143e8c16af4752fad7b40eb1085c014cbfcecb1ffc
```

**Review by alxgomz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gionn) [reply README.md]: to not use quay on public/opensource docker projects, but I guess it's already clear since I added the credentials requirements bit

**Review by pmacius — APPROVED** [review]

**Pull request merged**